src/ProductConstructionService/ProductConstructionService.BarViz/Pages/Build.razor

```diff
@@ -1,29 +1,108 @@
 ﻿@page "/build"
 @using ProductConstructionService.Client
 @using ProductConstructionService.Client.Models;
+@using TextCopy;
 @using ProductConstructionService.BarViz.Components;
 @inject IProductConstructionServiceApi PcsApi
+@inject IClipboard Clipboard
+@inject IJSRuntime JSRuntime
 
-<h3>Build</h3>
-@if (_build == null)
+@if (_build == null || _commit == null)
 {
     <p><em>Loading...</em></p>
 }
 else
 {
-    <h1>@_build.AzureDevOpsRepository</h1>
+    <div class="opposite">
+        <div>
+            <h4>
+                <span> @GetRepository() </span>
+                <span> @_build.AzureDevOpsBuildNumber </span>
+                <FluentButton IconEnd="@(new Icons.Regular.Size16.Copy())" Appearance="Appearance.Accent" OnClick="@(() => SetBarClipboard(_build.Id.ToString()))">
+                    @_build.Id
+                </FluentButton>
+            </h4>
+        </div>
+        <!-- TODO this should check if there's a failed or ongoing build that's supposed to go to this channel and display the info here -->
+        <FluentButton Appearance="Appearance.Accent" IconEnd="@(new Icons.Regular.Size16.Open())" OnClick="@(() => RedirectToUri(GetBuildUri()))">
```

Comment:
Why not just use `Href` and `Target="_blank"`? We would avoid all of this "open new window" stuff

Reply:
I couldn't find a FluentUI component that would do this. Maybe we could style an anchor to look like the button?

Reply:
maybe this `FluentNavLink`, I'll give it a try

Reply:
https://www.fluentui-blazor.net/Anchor

Reply:
it's not that great to be honest, I had the best results with the js that's in this PR

Reply:
What was not working with the Anchor?

Reply:
The anchor does work, but we'd need to style it to look consistent with Fluent UI

Reply:
Ah sorry I didn't see what you linked